Sources/Danger/BitBucketCloud.swift

```diff
@@ -0,0 +1,228 @@
+import Foundation
+
+public struct BitBucketCloud: Decodable {
+    /// The activities such as OPENING, CLOSING, MERGING or UPDATING a pull request
+    public let activities: [BitBucketCloudActivity]
+
+    /// The comments on the pull request
+    public let comments: [BitBucketCloudComment]
+
+    /// The commits associated with the pull request
+    public let commits: [BitBucketCloudCommit]
+
+    /// The pull request and repository metadata
+    public let metadata: BitBucketMetadata
+
+    /// The PR metadata
+    public let pr: BitBucketCloudPR
+}
+
+public struct BitBucketCloudPR: Decodable {
+    public enum State: String, Decodable {
+        case declined = "DECLINED"
+        case merged = "MERGED"
+        case open = "OPEN"
+        case suspended = "SUPERSEDED"
+    }
+
+    private enum CodingKeys: String, CodingKey {
+        case author
+        case createdOn = "created_on"
+        case description
+        case destination
+        case id
```

Comment:
<!--
  0 failure: 
  1 warning:  Enum element name...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/BitBucketCloud.swift;
  Line: 33;
-->

- :warning: Enum element name should be between 3 and 40 characters long: 'id' (`identifier_name`)